test/strings/types.jl

```diff
@@ -110,8 +125,6 @@ let s="lorem ipsum",
                SubString(s,1,6)=>"lorem ",
                SubString(s,1,0)=>"",
                SubString(s,2,4)=>"ore",
-               SubString(s,2,16)=>"orem ipsum",
```

Comment:
Maybe keep these examples, but with valid indices that still give the expected result?

Reply:
Done.